lib/Dialect/Lattigo/IR/LattigoBGVOps.td

```diff
@@ -97,7 +97,7 @@ class Lattigo_BGVBinaryOp<string mnemonic> :
   let arguments = (ins
     Lattigo_BGVEvaluator:$evaluator,
     Lattigo_RLWECiphertext:$lhs,
-    Lattigo_RLWECiphertext:$rhs
+    AnyType:$rhs
```

Comment:
Could we make this slightly more constrained than AnyType? I think this was changed because it's a union of ciphertext or plaintext type?